src/twisted/mail/imap4.py

```diff
@@ -223,8 +223,7 @@ def last(self):
         largest number in use.  Must be set before attempting to
         use the MessageSet as a container.
 
-        @raises: L{ValueError} if a largest value has already
-        been set.
+        @raises ValueError: if a largest value has already been set.
```

Comment:
well... maybe move the docs to setter?

But I think that is best to keep all the property docs includding setter docstring inside the main getter method.

I would say that this is a defect in pydoctor, as pydoctor should observer that we have a property and allow such a docstring..

if we don't have a setter pydoctror can highlight the property as read-only  ... otherwise it will be default read/write.

Reply:
I think it makes sense to have separate docstrings in the source for getters and setters, but to merge them in the output. I'll move it to the setter.

I agree that the presentation of properties in pydoctor is not ideal, but it is an improvement over ignoring them completely and I didn't want to delay the release even more.